contracts/Pool.sol

```diff
@@ -416,6 +416,7 @@ library ReceiverWeightsImpl {
         address receiver,
         uint32 weight
     ) internal returns (uint32 previousWeight) {
+        require(receiver > address(1), "Invalid receiver address");
```

Comment:
Why isn’t this `receiver != address(0)`?

Reply:
Because `ADDR_END = address(1)` and setting a weight to this address is an error too